lxd/api.go

```diff
@@ -81,7 +81,7 @@ func restServer(d *Daemon) *http.Server {
 			w.Header().Set("Permissions-Policy", "interest-cohort=()")
 			w.Header().Set("X-Content-Type-Options", "nosniff")
 			w.Header().Set("X-Frame-Options", "SAMEORIGIN")
-			w.Header().Set("X-Xss-Protection", "1; mode=block")
+			w.Header().Set("Content-Security-Policy", "default-src 'self' data: https://assets.ubuntu.com https://cloud-images.ubuntu.com; script-src 'self' 'unsafe-inline'; style-src 'self' 'unsafe-inline'")
```

Comment:
If we anticipate linking/using assets from other domains, we could also go with a domain wildcard like this:
```suggestion
			w.Header().Set("Content-Security-Policy", "default-src 'self' data: https://*.ubuntu.com; script-src 'self' 'unsafe-inline'; style-src 'self' 'unsafe-inline'")
```
It weakens the security a little but the added flexibility might be desirable.

Reply:
fixed